x/auth/migrations/v5/migrate_test.go

```
@@ -7,7 +7,8 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"cosmossdk.io/collections"
-	"cosmossdk.io/core/testing"
+	coretesting "cosmossdk.io/core/testing"
```

golangci-lint (1.62.2)

File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(cosmossdk.io) -s prefix(github.com/cosmos/cosmos-sdk) --custom-order

(gci)